synapse/rest/client/v2_alpha/report_event.py

```diff
@@ -58,17 +94,250 @@ async def on_POST(self, request, room_id, event_id):
                 Codes.BAD_JSON,
             )
 
+        target = body.get("org.matrix.msc2938.target", "homeserver-admins")
```

Comment:
experimental features like this should normally be gated by an `experimental_features` setting: see https://github.com/matrix-org/synapse/blob/develop/synapse/config/experimental.py.